roxctl/common/flags/endpoint_test.go

```diff
@@ -0,0 +1,85 @@
+package flags
+
+import (
+	"testing"
+
+	"github.com/stackrox/rox/pkg/env"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestEndpointAndPlaintextSetting(t *testing.T) {
+	truth := false
+	endpointChanged = &truth
+	plaintextSet = &truth
```

Comment:
Great prank. If you can't inline a value, please please name it differently, maybe `testFlagValue` or `flagValueForTest` or something like this.

Reply:
fixed with pointers util